etc/html/HELP.html

```diff
@@ -101,7 +101,7 @@ <h3><a id="module_libdep"></a>Modules' Library Dependencies</h3>
 <a href="./LibraryDependencies.html">Library Dependencies Chart</a>.
 
 <h2><a id="class"></a>Class Reference</h2>
-Pages like the one for <a href="http://root.cern.ch/root/html/TH2.html">TH2</a> are
+Pages like the one for <a href="http://root.cern/root/html/TH2.html">TH2</a> are
```

Comment:
Link should be replaced by https://root.cern/doc/master/classTH2.html

Reply:
Actually, http://root.cern/root/html/TH2.html also works